rio_tiler_pds/modis/aws/__init__.py

```diff
@@ -0,0 +1,4 @@
+"""rio-tiler-pds.modis.aws"""
+
+from .modis_astraea import MODISReader as MODISASTRAEAReader  # noqa
+from .modis_pds import MODISReader as MODISPDSReader  # noqa
```

Comment:
I'm not 💯 sure about the name of each Reader.

Basically there are 2 MODIS PDS on AWS,
1. (old) managed by nobody (used to be by devseed/astro-digital) but is in a Public Bucket
2. (new) managed by Astraea but in a requester-pays bucket.

The way the Reader are wrote, we can support both bucket in one Reader so we have to split in 2.